cookbooks/aws-parallelcluster-platform/kitchen.platform-config.yml

```diff
@@ -149,4 +149,15 @@ suites:
       - recipe[aws-parallelcluster-platform::networking]
     verifier:
       controls:
-        - /tag:config_networking/
\ No newline at end of file
+        - /tag:config_networking/
+  - name: sudo_configured
```

Comment:
nit: you can avoid the word "configured" since it's in the platform-config file.

Reply:
Sure.